sbol2/componentdefinition.py

```diff
@@ -1,6 +1,8 @@
 from rdflib import URIRef
 
+
```

Comment:
Does this extra line cause a pycodestyle warning?

Reply:
It didn't actually but I cleaned it up in d52c566